### PR TITLE
Enzyme tests

### PR DESCRIPTION
**Pull request by jeremy8883**
All components are done, except for "login", since it uses the redux store, and I have to figure out the best way to test these.

Connected to rangle/rangle-starter#72

### CODE REVIEW

**Comment by jeremy8883** [comment]
There were some linting errors because "isVisible={true}" existed on some components that are hidden by default. Would it be worth removing jsx-boolean-value from the linting process? Or should we have a rule that all components are visible by default (I prefer this one)?

**Comment by jeremy8883** [comment]
Sorry, fixing the coverage error...

**Comment by jeremy8883** [comment]
Fixed now

File: package.json [hunk 1]
```diff
@@ -73,7 +73,6 @@
     "html-webpack-plugin": "^2.9.0",
     "immutable": "^3.7.5",
     "istanbul": "^1.0.0-alpha",
-    "jsdom": "^8.3.0",
```
Comment (JaKXz) [inline package.json]: 🎉

**Comment by JaKXz** [comment]
love it [at first glance] - commits should probably be squashed though :)

**Comment by JaKXz** [comment]
Also, @jeremy8883 et al, you could [use closing keywords](https://github.com/waffleio/waffle.io/wiki/FAQs#prs-closing-keywords) in the PR description and the issue will be closed when this gets merged.

File: src/components/counter/index.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
 import React from 'react';
-import Button from './button';
+import Button from '../button';
 
 function Counter({ counter, increment, decrement }) {
   return (
     <div className="flex">
-      <Button className="bg-black col-2"
+      <Button data-ref="decrementButton" className="bg-black col-2"
```
Comment (SethDavenport) [inline src/components/counter/index.js]: In the past, we've adopted the convetion of using `id=qa-decrement-button` for stuff like this. I'm not opposed to `data-ref`, but the E2E tests are already using the id convention.

**Pull request merged**